src/data-browser/geo/MenuList.jsx

```diff
@@ -3,18 +3,46 @@ import { FixedSizeList as List } from 'react-window'
 
 const height = 35
 
-const MenuList = props => {
-  const { children, maxHeight } = props
-
-  return (
-    <List
-      height={maxHeight}
-      itemCount={children.length}
-      itemSize={height}
-    >
-      {({ index, style }) => <div style={style}>{children[index]}</div>}
-    </List>
-  )
+export class MenuList extends React.Component {
+  constructor(props) {
+    super(props)
+    this.listRef = undefined
+  }
+
+  getFocusIndex = (children) => {
+    const isArray = children instanceof Array
+    children = isArray ? children : [children]
+    return Math.max(
+      children.findIndex(({ props: { isFocused } = {} } = {}) => {
```

Comment:
findIndex will have to be replaced with a small for loop, as it's unsupported in ie